adr/README.md

```diff
@@ -0,0 +1,20 @@
+# Architecture Decision Records Folder
+This folder contains EdgeX Foundry architecture decision records (ADR).  At the root of the folder are decisions that are relevant to multiple parts of the project (aka � *cross cutting concerns*).  Sub folders contain decisions relevant to the specific area of the project and essentially set up along working group lines (security, core, application, etc.).
+## Naming and Formatting
+ADR documents are requested to follow [RFC]( https://tools.ietf.org/html/rfc2026) (request for comments) naming standard.  Specifically, authors should name their documents with a sequentially increasing integer (or serial number).  
```

Comment:
Clarification question -- is the ADR file named ####_Some_Description, Some_Description_####, ADR_####_Some_Description, or ...?

Reply:
I think the name needs to lead with the number, i.e. ####_Some_Description. Don't think we need ADR in the name since they are in the `adr` folder.

Reply:
I agree with @lenny-intel above but that's beside the point.   I'm just asking the question above in an attempt to minimize the confusion.